arch/RISCV/RISCVMapping.c

```diff
@@ -22,6 +24,49 @@
 static const name_map reg_name_maps[] = {
```

Comment:
Please remove this register name map. You should always use the `get_register_name` function provided by the LLVM code.